5_People/career-technical.md

```diff
@@ -6,40 +6,41 @@ The top two positions are two branches of the technical career:
 * Lead Software Engineer careers means you start taking over managerial tasks and leading people.
 * Principal Software Engineer means you take ownership of all of technical nitty-gritty details in the company. 
 
-## Junior Programmer
+## Junior Programmer / Sysadmin
 
 * Work is thoroughly reviewed with substantial back and forth frequently needed before merging.
-* Basic language features are mastered, but some advanced structures may still be unfamiliar.
+* Basic language/technology features are mastered, but some advanced approaches may still be unfamiliar.
 * Occasional issues following patterns and approaches within existing code bases.
 * Works mostly on clearly defined and scoped individual features or problems.
-* Usually less than 2 years of programming experience.
+* Usually less than 2 years of technical experience.
 
 
-## Junior Software Engineer
+## Junior Software Engineer / DevOps
```

Comment:
Could be `Junior DevOps`? Fine either way though.

Reply:
@zupo I would agree here with Niko. The title can't be the same for two roles.